File: MODUL02/PraktikumPy/PRAK205-2310817110004-AbdurrahmanGilangHarjuna.py
import math

tinggi = float(input("Sisi A = "))
samping = float(input("Sisi B = "))
alas = math.sqrt(samping ** 2 - tinggi ** 2)

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {alas + tinggi + samping:.0f} cm")
print(f"Luas = {0.5 * alas * tinggi:.0f} cm^2")

File: MODUL02/PraktikumC/PRAK205-2310817110004-AbdurrahmanGilangHarjuna.c
#include <stdio.h>
#include <math.h>

int main() {
    double tinggi, samping, alas;

    printf("Masukkan tinggi segitiga: ");
    scanf("%lf", &tinggi);
    printf("Masukkan samping segitiga: ");
    scanf("%lf", &samping);

    alas = sqrt(samping * samping - tinggi * tinggi);
    printf("Alas = %.lf cm\n", alas);
    printf("Tinggi = %.lf cm\n", tinggi);
    printf("Keliling = %.lf cm\n", alas + tinggi + samping);
    printf("Luas = %.lf cm^2\n", 0.5 * alas * tinggi);

    return 0;
}
